packages/filebrowser/src/model.ts

```diff
@@ -521,35 +519,32 @@ class FileBrowserModel implements IDisposable {
   }
 
   /**
+   * Start the internal refresh timer.
+   */
+  private _startTimer(): void {
+    this._timeoutId = window.setInterval(() => {
+      if (this._requested) {
+        return this.refresh();
```

Comment:
`return` on separate line, since the return value of `this.refresh()` means nothing here.